packages/features/data-table/lib/context.tsx

```diff
@@ -30,18 +30,22 @@ export type DataTableContextType = {
 
 export const DataTableContext = createContext<DataTableContextType | null>(null);
 
+const DEFAULT_ACTIVE_FILTERS: ActiveFilter[] = [];
+const DEFAULT_SORTING: SortingState = [];
+const DEFAULT_COLUMN_VISIBILITY: VisibilityState = {};
+
 export function DataTableProvider({ children }: { children: React.ReactNode }) {
   const [activeFilters, setActiveFilters] = useQueryState(
     "activeFilters",
-    parseAsArrayOf(parseAsJson(ZActiveFilter.parse)).withDefault([])
+    parseAsArrayOf(parseAsJson(ZActiveFilter.parse)).withDefault(DEFAULT_ACTIVE_FILTERS)
   );
   const [sorting, setSorting] = useQueryState(
     "sorting",
-    parseAsArrayOf(parseAsJson(ZSorting.parse)).withDefault([])
+    parseAsArrayOf(parseAsJson(ZSorting.parse)).withDefault(DEFAULT_SORTING)
   );
   const [columnVisibility, setColumnVisibility] = useQueryState<VisibilityState>(
     "cols",
-    parseAsJson(ZColumnVisibility.parse).withDefault({})
+    parseAsJson(ZColumnVisibility.parse).withDefault(DEFAULT_COLUMN_VISIBILITY)
```

Comment:
Previously this provided a new object or array every time it's called. So I created variables and use them as default values.

```
useEffect(() => {

}, [sorting]);
```

This kind of code have been executed unnecessarily due to this. Not a big perf win, but a small easy win :)

Reply:
Hey, everything counts 🙌